internal/lookout/repository/query.go

```diff
@@ -50,3 +58,285 @@ func (r *SQLJobRepository) GetQueueStats() ([]*lookout.QueueInfo, error) {
 	}
 	return result, nil
 }
+
+func (r *SQLJobRepository) GetJobsInQueue(opts *lookout.GetJobsInQueueRequest) ([]*lookout.JobInfo, error) {
+	rows, err := r.queryJobsInQueue(opts)
+	if err != nil {
+		return nil, err
+	}
+
+	result := parseJobsInQueueRows(rows)
+	return result, nil
+}
+
+type jobsInQueueRow struct {
+	JobId     string
+	Queue     string
+	Owner     string
+	JobSet    string
+	Priority  sql.NullFloat64
+	Submitted pq.NullTime
+	Cancelled pq.NullTime
+	JobJson   sql.NullString
+	RunId     sql.NullString
+	Cluster   sql.NullString
+	Node      sql.NullString
+	Created   pq.NullTime
+	Started   pq.NullTime
+	Finished  pq.NullTime
+	Succeeded sql.NullBool
+	Error     sql.NullString
+}
+
+func (r *SQLJobRepository) queryJobsInQueue(opts *lookout.GetJobsInQueueRequest) ([]*jobsInQueueRow, error) {
+	query, args, err := makeGetJobsInQueueQuery(opts)
+	if err != nil {
+		return nil, err
+	}
+
+	rows, err := r.db.Query(query, args...)
+	if err != nil {
+		return nil, err
+	}
+
+	joinedRows := make([]*jobsInQueueRow, 0)
```

Comment:
When we are using goqu now, maybe we should utilise the scanning functionality as well (https://github.com/doug-martin/goqu/blob/master/docs/selecting.md#scan-structs) ?